src/axom/sidre/spio/IOManager.hpp

```diff
@@ -289,11 +289,47 @@ class IOManager
    * This currently only works if the root file was created for protocol
    * sidre_hdf5.
    *
+   * The call to this must follow a call to the IOManager::read with the
+   * same group and root file.
+   *
+   * This is intended as the third step of the three step process to load
+   * external data.  The first step is the call to IOManager::read; the
+   * second step is to set valid pointers on all external views in the
+   * hierarchy under the passed-in group; the third step is the call to
+   * this method.
+   *
    * \param group         Group to fill with external data from input
    * \param root_file     root file containing input data
    */
   void loadExternalData(sidre::Group* group, const std::string& root_file);
 
+  /*!
+   * \brief piecewise load of external data into a group
+   *
+   * This currently only works if the root file was created for protocol
+   * sidre_hdf5.
+   *
+   * This is intended as an alternative way to load external data, in a
+   * piecewise manner rather than loading all external data of a Sidre
+   * hierarchy at once.  load_group is a Group somewhere in the hierarchy
+   * under parent_group, and only the external data for views in the
```

Comment:
why is parent group needed here?

Reply:
In order to find the needed data in the HDF file, the method needs the relative path from parent_group to load_group.  The logic to figure this out is inside the method.  The alternative would be to force the user to construct this relative path and pass it in.